test/__snapshots__/build-index-errors.test.js.snap

```diff
@@ -1,5 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`build-index-errors empty-routes 1`] = `[Error: Empty \`routes/\` directory at test/build-index-errors/empty-routes/routes]`;
+exports[`build-index-errors empty-routes 1`] = `[Error: Expected a \`route.js\`, template, or \`routes/\` directory at test/build-index-errors/empty-routes/routes/file]`;
```

Comment:
I would prefer that we test the old error here.  `routes-table` is picking up the `file` I added to ensure that the directory got committed and treating it as a route rather than seeing `routes/` as an empty directory.

Would you try renaming `file` to `.file`?